orm/model/ormdb/module.go

```diff
@@ -29,13 +29,17 @@ type ModuleDB interface {
 	// GenesisHandler returns an implementation of appmodule.HasGenesis
 	// to be embedded in or called from app module implementations.
 	// Ex:
-	//   type Keeper struct {
+	//   type AppModule struct {
```

Comment:
Not doing like this, I was getting a depinject issue in https://github.com/cosmos/cosmos-sdk/pull/15304 (https://github.com/cosmos/cosmos-sdk/pull/15304/commits/37d2d621fa6f3f37340a50c5873d840c3860cd9c)